flow/SystemMonitor.cpp

```diff
@@ -51,6 +51,17 @@ SystemStatistics customSystemMonitor(std::string eventName, StatisticsState *sta
 	NetworkData netData;
 	netData.init();
 	if (!DEBUG_DETERMINISM && currentStats.initialized) {
+		int64_t unusedMemory = 0;
+		unusedMemory += FastAllocator<16>::getMemoryUnused();
+		unusedMemory += FastAllocator<32>::getMemoryUnused();
+		unusedMemory += FastAllocator<64>::getMemoryUnused();
+		unusedMemory += FastAllocator<128>::getMemoryUnused();
+		unusedMemory += FastAllocator<256>::getMemoryUnused();
+		unusedMemory += FastAllocator<512>::getMemoryUnused();
+		unusedMemory += FastAllocator<1024>::getMemoryUnused();
+		unusedMemory += FastAllocator<2048>::getMemoryUnused();
+		unusedMemory += FastAllocator<4096>::getMemoryUnused();
```

Comment:
Is there any way of enumerating these other than writing them all out like this? In particular, is there any way that we will notice that we missed one if we added, say, a FastAllocator<8192>?